src/app/shared/components/template/components/layout/nav_group.ts

```diff
@@ -118,14 +133,18 @@ export class NavGroupComponent extends TemplateLayoutComponent {
     }
     if (action_id === "emit" && args[0] === "uncompleted") {
       if (this.sectionIndex > 0) {
-        this.goToSection(this.sectionIndex - 1);
+        this.goToSection(this.sectionIndex - 1).then((r) => console.log(r));
```

Comment:
Not convinced we want this but don't feel strongly

Reply:
Function 'goToSection' is async function , it will be good practice to write down 'then' part. 
But for a timebeing , I will remove the inner part.